src/main/java/seedu/address/model/task/Task.java

```diff
@@ -36,6 +36,10 @@ public String getModule() {
         return this.module.toString();
     }
 
+    public ModuleName getModuleName() {
+        return this.module;
+    }
```

Comment:
Maybe I can suggest changing the ModuleName field in task from `module` to `moduleName`?

Reply:
yep that's better